cross-modal-search/flows/flow-index.yml

```diff
@@ -6,13 +6,15 @@ with:
   workspace: $JINA_WORKSPACE
 pods:
   - name: loader                                      # load images from the dataset of image-caption pairs
+    # uses: 'jinahub+docker://ImageNormalizer'
     uses: pods/image-load.yml
     shards: $JINA_PARALLEL
     read_only: true
     needs: [gateway]
   - name: image_encoder                               # encode images into embeddings with CLIP model
-    uses: 'jinahub+docker://ClipImageEncoder'
-    volumes: $HOME/.cache/clip:/root/.cache/clip
+    # uses: 'jinahub+docker://CLIPImageEncoder'
+    # volumes: $HOME/.cache/clip:/root/.cache/clip
```

Comment:
remove the unused codes

Reply:
Addressed, please see [e11b980](https://github.com/jina-ai/examples/pull/672/commits/e11b980488645a32eac1fbe3a7c9ca27def8d850) for the change.